src/record/export/usecases/get.ts

```diff
@@ -111,7 +112,9 @@ const fieldProcessor: (
           const localFilePath = path.join(
             attachmentsDir,
             `${fieldSchema.code}-${recordId}`,
-            fileInfo.name
+            process.platform === "win32"
+              ? sanitizeFilename(fileInfo.name)
+              : fileInfo.name
```

Comment:
MUST: You need to change the same process of the `fieldProcessorInSubtable()` too.

Reply:
Fixed with 42bcc409cfa1b68bf2a8b11eb8d31950c7836fde